crates/holochain_conductor_api/src/config.rs

```diff
@@ -1,6 +1,8 @@
 use serde::Deserialize;
 use serde::Serialize;
 
+pub mod conductor;
+
 /// Information neeeded to spawn an Admin interface
```

Comment:
I'm OK with this, but maybe everything else in this file should go into an `interface` subdir so it lives alongside `conductor`?